JavaScript/packages/recognizers-sequence/src/sequence/extractors.ts

```diff
@@ -214,13 +220,22 @@ export class BaseEmailExtractor extends BaseSequenceExtractor {
 
 export class BaseURLExtractor extends BaseSequenceExtractor {
     regexes: Map<RegExp, string>;
+    ambiguousTimeTerm: RegExp;
 
     constructor(){
         super();
         this.regexes = new Map<RegExp, string>()
             .set(RegExpUtility.getSafeRegExp(BaseURL.UrlRegex), Constants.URL_REGEX)
             .set(RegExpUtility.getSafeRegExp(BaseURL.UrlRegex2), Constants.URL_REGEX)
             .set(RegExpUtility.getSafeRegExp(BaseURL.IpUrlRegex), Constants.URL_REGEX)
+        this.ambiguousTimeTerm = RegExpUtility.getSafeRegExp(BaseURL.AmbiguousTimeTerm);
+    }
+
+    isValidMatch(match: Match): Boolean {
+        if (RegExpUtility.isMatch(this.ambiguousTimeTerm, match.value)) {
```

Comment:
Nit: We could just directly return the negation of the condition. Code is a little cleaner that way.